### PR TITLE
Fix #392 (case of lack of reserched terms)

### LINKED ISSUE #392: Software status filter in search page is broken
By selecting more than one option of software status filter in search page, 0 softwares are shown, even if they match the selected criteria. 

An `undefined` is given as searched word.
Pagination footer seems broken as well.

Screenshot 2019-09-03 at 15 25 44

### PR DESCRIPTION
**Pull request by ffranchina**
Hello, this is PR solves the above issue.
I also suggested a slightly different translation for the english part, I hope it's fine.
Thanks for your time!

### CODE REVIEW

File: assets/scripts/esDevelopersItaliaQuery.js [hunk 1]
```diff
@@ -508,12 +508,12 @@ function esDevelopersItaliaQuery(config, params) {
     'emptySerp': {
       'it': {
         'title': 'Nessun risultato trovato',
-        'message': 'La parola "{keyword}" non ha prodotto nessun risultato. Prova un\'altra chiave di ricerca.',
+        'message': 'La parola cercata non ha prodotto nessun risultato. Prova un\'altra chiave di ricerca.',
```
Comment (libremente) [inline assets/scripts/esDevelopersItaliaQuery.js]:
```suggestion
        'message': 'La ricerca non ha prodotto nessun risultato. Prova un\'altra chiave di ricerca.',
```

File: assets/scripts/esDevelopersItaliaQuery.js [hunk 2]
```diff
@@ -508,12 +508,12 @@ function esDevelopersItaliaQuery(config, params) {
...
         'cta': 'Prova una nuova ricerca'
       },
       'en': {
         'title': 'No results found',
-        'message': 'The word "{keyword}" did not find any results. Try another search key.',
+        'message': 'The searched terms did not match any results. Try searching other terms.',
```
Comment (libremente) [inline assets/scripts/esDevelopersItaliaQuery.js]:
```suggestion
        'message': 'The search did not match any results. Try searching other terms.',
```

**Comment by ffranchina** [comment]
Thanks! I guess it should be fine now :)

**Comment by libremente** [comment]
Thanks @ffranchina! Merged 🎉

**Pull request merged**